src/main/java/anab/Person.java

```diff
@@ -0,0 +1,46 @@
+public class Person {
+
+  String FirstName;
```

Comment:
please change the name of this variable to firstName

Reply:
Changes done under commit https://github.com/alexandruandriesei/svPreEmp/pull/57/commits/d4191ddaba8581bc639147c46207ab0eac419df3